src/docs/src/whatsnew/3.4.rst

```diff
@@ -39,6 +39,14 @@ Breaking Changes
   We recommend adding ``nbf`` as a required claim if you know your tokens will
   include it.
 
+* :ghissue:`5203`: Continuous change feeds with ``descending=true&limit=N``
+
+  Previously, ``_changes?feed=continuous&descending=true&limit=N`` with ``N``
+  greater than total rows would block and wait on database changes. On each
+  change the first few sequence rows (sequences ``1-...``, ``2-...``) were
+  repeatedly emitted. The new behavior is to return after all the rows are
```

Comment:
The description of the new behaviour needs some work. will ponder over the weekend. We want to convey that the (internal) phase 2 of live updates will not be run (as they obviously can't be in the requested order).

That is, we recognise that feed=continuous and descending=true are incompatible, and have chosen to honour the second setting (all the rows returned in the response are in descending order). as we know that any new update would belong at the beginning of the response, which is now long past, we choose to end the response.

Reply:
I tried another version. I think we want to emphasize that the previous behavior was non-sensical and downright broken, not as much that we picked one another option vs another, almost as reasonable, option.

In other words, I could see viewing it as a trade-off if we had actually emitted the newly updated row in phase 2. Something like: `3, 2, 1, (phase2)... 4, 5, 6`.  Instead now it seems to be `3,2,1, (phase2)... 2,1,2,1...`.